### ISSUE TITLE
Gender confusion

### ISSUE THREAD

**Issue by RonTampon** [gbiervliet/teentjestellen#1]
![did-you-just-assume-my-gender](https://user-images.githubusercontent.com/5338169/33739386-23b41894-db9d-11e7-9dc1-7bf549b1fe91.jpg)

https://github.com/gbiervliet/teentjestellen/blob/70424a1a7666cebf1773420e97f30cd14883157f/src/database.php#L22-L27
When a person is not voting for a male, you are amusing _it_ is a female (id 1). This is not correct since there are at least 71 genders.

**Comment by RonTampon** [comment]
#MeToo

**Comment by gbiervliet** [comment]
@RonTampon you are absolutely right .. or left .. *not assuming* but I don't know how to think in a non-binary way. Can you please create a pull request ?
